pkg/middlewares/aws_instance_block_device.go

```diff
@@ -28,6 +32,110 @@ func (a AwsInstanceBlockDeviceResourceMapper) Execute(remoteResources, resources
 			continue
 		}
 
+		if resource.IsRefactoredResource("aws_instance") {
+			instance, _ := stateRes.(*resource.AbstractResource)
+
+			if rootBlockDevice, exist := instance.Attrs.Get("root_block_device"); exist {
+				for _, rootBlock := range rootBlockDevice.([]interface{}) {
+					rootBlock := rootBlock.(map[string]interface{})
+					logrus.WithFields(logrus.Fields{
+						"volume":   rootBlock["volume_id"],
+						"instance": instance.TerraformId(),
+					}).Debug("Creating aws_ebs_volume from aws_instance.root_block_device")
+					data := map[string]interface{}{
+						"availability_zone": func() interface{} {
```

Comment:
This will work no ? `"availability_zone": instance.Attrs["availability_zone"]`

Reply:
It will be `"availability_zone": (*instance.Attrs)["availability_zone"],`

Reply:
Yeah, maybe it's more straightforward ?